build.gradle

```diff
@@ -904,69 +436,35 @@ publishing {
 			}
 			artifact sourcesJar
 			artifact javadocJar
-		}
 
+			pom.withXml {
+				def hashedDep = it.asNode().appendNode('dependencies').appendNode('dependency')
+				hashedDep.appendNode('groupId', 'org.quiltmc')
+				hashedDep.appendNode('artifactId', 'hashed')
```

Comment:
```suggestion
				hashedDep.appendNode('artifactId', 'hashed-mojmap')
```

Reply:
the namespace in tiny is hashed, not hashed-mojmap.

Reply:
Isn't this supposed to point to an artifact?

Reply:
true. I didn't think of that. @CheaterCodes do we want to either move the mapping namespace to hashed-mojmap, or move the artifact to hashed

Reply:
I'd say moving the artifact is the best option